en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/P/Pelhams_Hovering_Platter.md

```diff
@@ -13,7 +13,8 @@
 [_metadata_:components_material_description]:- "a drop of mercury"
 [_metadata_:concentration]:- "false"
 [_metadata_:duration]:- "1 hour"
-[_metadata_:spell_origin]:- "wotc_srd_5.1"
+[_metadata_:compared_to_wotc_srd]:- "mechanics_same_wording_different"
+[_metadata_:compared_to_a5e_srd]:- "mechanics_different_wording_different"
```

Comment:
The different mechanics in this case are very small, but relevant in some very specific cases: A5e makes this a metallic platter. It could thereby be subject to things like _Heat Metal_.